### ISSUE TITLE
onRequestUploadTarget is not a function

### ISSUE THREAD

**Issue by ailet37** [avocode/avocode#1110]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.10.1`
Source UID: 91877
```
TypeError: this.props.onRequestUploadTarget is not a function
    at Constructor._handleDragEnter (file:///C:/Users/verstak1/AppData/Local/avocode/app-2.10.1/resources/app.asar/dist/avocode.min.js:21:7404)
    at Immediate.<anonymous> (C:\Users\verstak1\AppData\Local\avocode\app-2.10.1\resources\app.asar\node_modules\react-droparea\index.js:106:18)
    at runCallback (timers.js:566:20)
    at tryOnImmediate (timers.js:546:5)
    at processImmediate [as _immediateCallback] (timers.js:525:5)
```

**Comment by connor-bean** [comment]
Hey @ailet37 -

Could you provide us with more information as to what you were doing when you received this error?

Thanks,
Connor, Avocode Support

**Comment by petrbrzek** [comment]
@connor-bean I'm able to reproduce the bug.

**Comment by connor-bean** [comment]
We just released 2.11 so I'm closing this out. Please let us know if this problem persists.

You can download the update at https://avocode.com/news.html

Thanks for your patience on this.

Connor, Avocode Support
